package.json

```diff
@@ -166,7 +166,8 @@
     "babel-jest": "workspace:*",
     "jest": "workspace:*",
     "jest-environment-node": "workspace:*",
-    "react-native@0.68.1": "patch:react-native@npm:0.68.1#.yarn/patches/react-native-npm-0.68.1-8830b7be0d.patch"
+    "react-native@0.68.1": "patch:react-native@npm:0.68.1#.yarn/patches/react-native-npm-0.68.1-8830b7be0d.patch",
+    "@types/node": "~12.12.0"
```

Comment:
Perhaps `@types/node` should be first in this list? To keep order similar to `devDependencies`?

Reply:
good call!